integrations/jenkins/.port/resources/port-app-config.yaml

```diff
@@ -0,0 +1,43 @@
+createMissingRelatedEntities: true
+deleteDependentEntities: true
+resources:
+  - kind: job
+    selector:
+      query: "true"
+    port:
+      entity:
+        mappings:
+          identifier: .fullName | gsub(" "; "") | gsub("/"; "-") | ascii_downcase
```

Comment:
why lowercase?

Reply:
for consistency in the identifier format.